mrc/configuration/configurator.py

```diff
@@ -0,0 +1,21 @@
+from mrc.shared.position import Position
+
+
+class Configurator:
+    def __init__(self, id_to_location):
```

Comment:
I thought that it would be good to allow user to use either dictionary or a function that would work exactly the same

Reply:
Nice. But "id_to_location" is, in my opinion, a pretty bad name. It took me a while to realize what it's supposed to do ;)

Reply:
Name changed